configure.ac

```diff
@@ -184,10 +185,10 @@ fi
 ##############################################################################
 # GNU Scientific Library
 
-AC_CHECK_FUNC(cblas_cgemm, [], [AC_CHECK_LIB(gslcblas, cblas_cgemm)])
-
-AC_CHECK_LIB(gsl, gsl_sf_bessel_Jn, [],
-	[AC_MSG_WARN([Missing GNU GSL library...Bessel-function field initialization will not be supported.])])
+AC_CHECK_LIB([m],[cos])
+AC_CHECK_LIB([gslcblas],[cblas_dgemm])
```

Comment:
This doesn't work as is, at least not on my system. Linking with `-lgsl` but not `-lgslcblas` fails with error messages of the form

```
/home/homer/codes/lib/libgsl.so: undefined reference to `cblas_dgemm'
```

This is true **even if** I explicitly link to a BLAS library that contains the `cblas_` routines. I have tested this both with openblas and with the standard system BLAS (`/usr/lib/libblas.{a,so}`). Both of these definitely do provide the `cblas` symbols. For example,

```
% nm --dynamic /home/homer/codes/lib/libopenblas.so | grep cblas_dgemm
0000000000093800 T cblas_dgemm
```

But linking with e.g. `... -lgsl -lopenblas ...` fails with the above error messages about not finding `cblas_dgemm` and friends.

On the other hand, if I add `-lgslcblas` then it works, even if I don't remove `-lopenblas` from the command line. What is surprising is that `libgslcblas.so` seems to be providing the same symbols that e.g. openblas provides, for example: 

```
2030 hikari /tmp <> nm --dynamic /usr/lib/libgslcblas.so | grep cblas_dgemm
000000000000e880 T cblas_dgemm
```

The fact that I can link both openblas and gslcblas at the same time despite the fact that they both define the same symbols means that the symbols defined by the latter must be weak symbols somehow, and the fact that the codes in libgsl refuse to accept the openblas versions of these symbols means that there must be some sort of tagging going on in the GSL build. I will look into this. 

For the moment, since we agree that linking to gslcblas works and is harmless except for introducing possibly spurious extra code mass, should we keep things the way they are?